### PR TITLE
Install and configure tooling

### PR DESCRIPTION
**Pull request by niklasnatter**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

This PR installs and configures our usual tooling stack:
* editorconfig file
* php-cs-fixer
* phpstan

### CODE REVIEW

File: Tests/Unit/EventListener/SetThemeEventListenerTest.php [hunk 1]
```diff
@@ -12,18 +13,18 @@
...
-use Sulu\Component\Webspace\Theme;
 use Sulu\Component\Webspace\Webspace;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 
 class SetThemeEventListenerTest extends TestCase
 {
     /**
-     * @var ActiveTheme
+     * @var ObjectProphecy>
```
Comment (alexander-schranz) [inline Tests/Unit/EventListener/SetThemeEventListenerTest.php]: Can you check this.

**Review by alexander-schranz — CHANGES_REQUESTED** [review]

File: EventListener/SetThemeEventListener.php [hunk 2]
```diff
@@ -38,7 +38,7 @@ public function __construct(ActiveTheme $activeTheme)
      *
      * @param GetResponseEvent $event
      */
-    public function setActiveThemeOnRequest(GetResponseEvent $event)
+    public function setActiveThemeOnRequest(GetResponseEvent $event): void
```
Comment (alexander-schranz) [inline EventListener/SetThemeEventListener.php:41]: sorry this is a bc break we can not do this if not really needed use phpdocs instead.

File: EventListener/SetThemeEventListener.php [hunk 3]
```diff
@@ -55,7 +55,7 @@ public function setActiveThemeOnRequest(GetResponseEvent $event)
      *
      * @param PreRenderEvent $event
      */
-    public function setActiveThemeOnPreviewPreRender(PreRenderEvent $event)
+    public function setActiveThemeOnPreviewPreRender(PreRenderEvent $event): void
```
Comment (alexander-schranz) [inline EventListener/SetThemeEventListener.php:58]: sorry this is a bc break we can not do this if not really needed use phpdocs instead.

File: EventListener/SetThemeEventListener.php (hunk 2 above)
Comment (niklasnatter) [reply EventListener/SetThemeEventListener.php:41]: 😢

Comment (niklasnatter) [reply EventListener/SetThemeEventListener.php:41]: i decided that im willing to make this break 😜

Comment (alexander-schranz) [reply EventListener/SetThemeEventListener.php:41]: 👿

**Pull request merged**